concourse/tasks/run-link-checker.yml

```diff
@@ -0,0 +1,29 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/cf-cli
+    tag: latest
+params:
+  CF_API: https://api.cloud.service.gov.uk
+  CF_USERNAME: ((paas-username))
+  CF_PASSWORD: ((paas-password))
+  CF_ORG: govuk_development
+  SENTRY_DSN: https://((sentry-dsn))
+  PUBLISHING_API_BEARER_TOKEN:
```

Comment:
Before we merge, I will need to add the bearer tokens to secrets.. this line will need to become:

```
PUBLISHING_API_BEARER_TOKEN: ((publishing-api-bearer-token))
```

The `(())` syntax means that it is being read in from GOV.UK secrets.